samples/databus/custom-serializer/Core_6/Shared/JsonDataBusSerializer.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Text;
+
+namespace Shared
+{
+    using System.IO;
+    using Newtonsoft.Json;
+    using NServiceBus.DataBus;
+
+    #region CustomDataBusSerializer
+
+    public class JsonDataBusSerializer : IDataBusSerializer
+    {
+        public void Serialize(object databusProperty, Stream stream)
+        {
+            using (var writer = new StreamWriter(stream, Encoding.UTF8, bufferSize: 1024, leaveOpen: true))
```

Comment:
Nah, I think it would hide the intent. How about this:
```
public void Serialize(object databusProperty, Stream stream)
{
    using (var writer = new StreamWriter(
        stream, Encoding.UTF8, bufferSize: 1024, leaveOpen: true))
    using (var jsonWriter = new JsonTextWriter(writer))
    {
        serializer.Serialize(jsonWriter, databusProperty);
    }
}
```
The line break doesn't look that nasty in my opinion.